packages/sdk-codegen/CHANGELOG.md

```diff
@@ -5,9 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [21.0.11](https://www.github.com/looker-open-source/sdk-codegen/compare/sdk-codegen-v21.0.10...sdk-codegen-v21.0.11) (2021-03-30)
+
+
```

Comment:
This *definitely* should have the `--versions` comment

Reply:
I tracked down the release-please bug: `packages/sdk` was overshadowing `packages/sdk-*` depriving them of getting an conventional commits assigned. Fixed it and now all 3 have the `Added --versions...` commit and got the `minor` bump.